### ISSUE TITLE
R CMD check failures

### ISSUE THREAD

**Issue by hadley** [berndbischl/ParamHelpers#57]
I see

```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.
```
```
checking tests ... ERROR
Running the tests in ‘tests/run-all.R’ failed.
Last 13 lines of output:
             "loadBalancing"))), .Names = c("no.candidates", "no.tasks", "race.instances", 
         "candidates", "parameters", "config")))
  15: execute.experiments(experiments, length(experiments), data$config)
  16: .irace$hook.run(experiments[[k]], config = configuration)
  
  Warning in generateDesign(1000, ps) :
    generateDesign could only produce 996 points instead of 1000!
  testthat results ================================================================
  OK: 768 SKIPPED: 0 FAILED: 1
  1. Error: convertParamSetToIrace 
  
  Error: testthat unit tests failed
  Execution halted
```

**Comment by larskotthoff** [comment]
I can't reproduce this (and the title field doesn't end in a period?). Tested with ParamHelpers HEAD on R version 3.2.2 and latest development version (2015-09-30 r69442) with CRAN and HEAD versions of `testthat`.

What versions/environment are you using?

**Comment by hadley** [comment]
The CRAN version of ParamHelpers. Maybe you've fixed it dev?

**Comment by larskotthoff** [comment]
Yep, there were some changes in the upstream package. This was fixed in 941c45d739a3cf6288720c0787bcb444626d2bff and a new version of ParamHelpers will be released soon.

Thanks for your report though.
